internal/sveltos/profile.go

```diff
@@ -224,3 +225,50 @@ func PriorityToTier(priority int32) (int32, error) {
 
 	return 0, fmt.Errorf("invalid value %d, priority has to be between %d and %d", priority, mini, maxi)
 }
+
+// SetStatusConditions transforms status from Sveltos ClusterSummary
+// object and sets it into the provided list of conditions.
+func SetStatusConditions(summary sveltosv1beta1.ClusterSummary, conditions *[]metav1.Condition) {
```

Comment:
can the function's signature be changed in order to have the `clustersummary` instance as a reference instead of a copy?